tools/SchemaManager/Utils/BaseSchemaRunner.cs

```diff
@@ -4,7 +4,7 @@
 // -------------------------------------------------------------------------------------------------
 
 using System;
-using Microsoft.Data.SqlClient;
+using System.Data.SqlClient;
```

Comment:
I am not sure about this change, can you check with Brendon

Reply:
Microsoft.Data.SqlClient is now used throughout the solution. I believe it is Microsoft.Data.SqlClient in the latest iteration.